### ISSUE TITLE
Quiz Page // Connect to API

### ISSUE THREAD

**Issue by alinarub** [alinarub/dog-app#9]
## Value proposition
**As a user** 
I want to get results from the API, not from the dummy data.

**In order** to have a better quality result.

## Description
![Image](https://github.com/alinarub/dog-app/assets/137082094/69806b3d-c097-41c7-8ffc-f4811b21f91d)
 
## Acceptance criteria
- [ ] Our calculation function `assignPointsToDog` still works as previously but now with the API.
- [ ] As a result, the matching dogs from the API are stored in a separate `assignedPoints` state. The state ranks the dogs.
 
## Tasks
- [ ] Create the feature branch `api-connect`.
- [ ] Connect the API to our dogPoints algorithm.

**Comment by NikRoe** [comment]
Hi there,

your users probably don't care where the data comes from. But they care to have better results. So I would rephrase the Value Proposition:

> As a user
I want to get real data
In order to have a better quality result.

In your Acceptance Criteria you want to describe the behaviour of the App.
Which could be something like:

> The quiz uses data from the API

The information regarding the assignedPoints state is a Tasks, not an AC

Keep it up 🙌
